src/browser/api_protocol/api_handlers/system.js

```diff
@@ -252,6 +253,12 @@ function SystemApiHandler() {
         ack(dataAck);
     }
 
+    function getEntityInfo(identity, message, ack, nack) {
+        const { uuid, name } = message.payload;
+
+        return System.getEntityInfo({ uuid, name });
```

Comment:
ack or nack?

Reply:
just the return is enough here, it will auto ack, or nack if it throws